controller_manager/src/controller_manager.cpp

```diff
@@ -553,6 +553,18 @@ controller_interface::return_type ControllerManager::configure_controller(
       controller_name, std::make_unique<ControllerThreadWrapper>(controller, update_rate_));
   }
 
+  const auto controller_update_rate = controller->get_update_rate();
+  const auto cm_update_rate = get_update_rate();
+  if (controller_update_rate > cm_update_rate)
```

Comment:
```suggestion
  if (controller_update_rate >= cm_update_rate)
```
As per the error message ;)

Reply:
I think it is correct as before right, we say that the controller update rate should be less than or can be equal to the controller manager's update rate. If it is equal, do we still have to print the warning?

Reply:
you are right, false alarm!